app/run.py

```diff
@@ -0,0 +1,138 @@
+import os
+import re
+import random
+import json
+from dotenv import load_dotenv
+from rake_nltk import Rake
+
+from flask import Flask
+from flask import request, jsonify
+from flask_cors import CORS
+
+import spacy
+import torch
+from transformers import GPT2LMHeadModel, GPT2Tokenizer, GPT2Config
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+
+
+from song_story import SpacyModel
+from song_story import TextRank
+from song_story import GPT2, T5
+from song_story import GeniusClient
+
+load_dotenv()
+
+
+class App(Flask):
+    def __init__(self, name):
+        super(App, self).__init__(name)
+
+        # Load models
```

Comment:
Don't include so much comments in between, anything which is straightforward can be understood with name of the variable or function used

Reply:
Removed the comments